### ISSUE TITLE
concept: please clearify

### ISSUE THREAD

**Issue by labor4** [PushBits/cli#7]
Hi

Interesting project.
But I don't understand the basic concept.

It appears each "application" can only have 1 user. Correct?

1. pbcli -u admin creates "user1"
2. pbcli -u user1 creates "app1" (token created)
3. user1 receives invitation via matrix
4. curl to pushbit-server (with token) sends message to user1 (via matrix)

It is a bit misleading that user1 becomes the authorized creator of the application, AS WELL as its sole recipient.
So basically user1 sends this to itself.

I expected user1 to be an app-creator (non-admin priv.), and application to hold multiple subscribers (a third role, with no password).

Not sure.

**Comment by eikendev** [comment]
Hi @labor4, PushBits allows you to set up your own notification system, enabling the applications, scripts, or tools *you use* to send messages directly *to you*. This makes sense if you create run, e.g., a server monitoring tool, a stock price tracker, or a custom home automation script.
